receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -112,13 +111,17 @@ func (s *scraper) scrape(_ context.Context) (pmetric.Metrics, error) {
 		if err = s.scrapeAndAppendDiskIOMetric(now, md.handle); err != nil {
 			errs.AddPartial(diskMetricsLen, fmt.Errorf("error reading disk usage for process %q (pid %v): %w", md.executable.name, md.pid, err))
 		}
-
-		s.mb.EmitForResource(md.resourceOptions()...)
+		options := append([]metadata.ResourceMetricsOption{metadata.WithStartTimeOverride(pcommon.Timestamp(start_time * 1e9))}, md.resourceOptions()...)
```

Comment:
Just for better readability :)

```suggestion
		options := append(md.resourceOptions(), metadata.WithStartTimeOverride(pcommon.Timestamp(start_time * 1e9)))
```